### PR TITLE
New environments documentation

### LINKED ISSUE #542: Setup new hosted environments for `testing-preview`, `pre-release-preview` and `release-preprod`) with their `terraform` and `GitHub environments`

### PR DESCRIPTION
**Pull request by jpraynaud**
## Content
This PR includes:
* A dev blog post explaining the evolution of the testing environments for SPOs
* The updated list of `Mithril Networks`

## Issue(s)
Relates to #500 or Closes #542

### CODE REVIEW

File: docs/blog/2022-10-28-updated-environments.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### The Mithril environments are updated
+
+**PR**: `New hosted environments` [#561](https://github.com/input-output-hk/mithril/pull/561)
+
+**Issue**: `Setup new hosted environments for testing-preview, pre-release-preview and release-preprod) with their terraform and GitHub environments` [#542](https://github.com/input-output-hk/mithril/issues/542)
+
+On Tuesday, November 1st, 2022 the `preview` Cardano network will be re-spun and will be unavailable for 48h.
+
+In the mean time, the Mitril team is also releasing a new Release Process that will make use of several new environments.
```
Comment (abailly-iohk) [inline docs/blog/2022-10-28-updated-environments.md:16]: releasing -> implementing

**Review by abailly-iohk — COMMENTED** [review]
Perhaps you could add a note explaining what's the future target? I think people could find the fact there's a  `release-preprod` and a `release-main` confusing. Saying something around the idea that the `release-preprod` will end up being a beta-testing or something?

File: docs/blog/2022-10-28-updated-environments.md (hunk 1 above)
Comment (jpraynaud) [reply docs/blog/2022-10-28-updated-environments.md]: Done :heavy_check_mark:

**Comment by jpraynaud** [comment]
> Perhaps you could add a note explaining what's the future target? I think people could find the fact there's a `release-preprod` and a `release-main` confusing. Saying something around the idea that the `release-preprod` will end up being a beta-testing or something?

I have added the note regarding the future target :+1:

**Review by iquerejeta — APPROVED** [review]
LGTM 👍

**Review by Alenar — APPROVED** [review]

**Pull request merged**